brayns/io/MeshLoader.h

```diff
@@ -39,7 +39,7 @@ namespace brayns
 class MeshLoader
 {
 public:
-    MeshLoader(GeometryParameters& geometryParameters);
+    MeshLoader();
```

Comment:
why removed? the geometry params are always the same, so giving them once in the construction is fine.

Reply:
MeshLoader is a class member of Brayns and we do not have the geometry parameters yet when the MeshLoader is created. I could add a setGeometryParameters to the MeshLoader class but this would not make much difference with the current implementation. What do you think?